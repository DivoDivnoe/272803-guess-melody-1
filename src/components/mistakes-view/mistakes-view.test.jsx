import React from 'react';
import renderer from 'react-test-renderer';
import MistakesView from './mistakes-view.jsx';

const mock = {
  mistakes: 5,
};

describe(`MistakesView component`, () => {
  it(`renders correctly`, () => {
    const tree = renderer.create(
        <MistakesView mistakes={mock.mistakes} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
