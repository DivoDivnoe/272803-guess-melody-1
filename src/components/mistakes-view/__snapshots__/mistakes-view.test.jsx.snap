// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MistakesView component renders correctly 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
</div>
`;
